how-to-build-mobile.md

```diff
@@ -0,0 +1,39 @@
+# Building Cryptopower Wallet for Mobile
+
+This readme assumes you have a working Android or iOS environment.
+
+## 1. Building for Android
+
+Note: You need to have [gogio](https://gioui.org/doc/install/android) to build.
+
+execute the command below in a terminal window to install gogio:
+
+`go install gioui.org/cmd/gogio@latest`
+
+cd to the cryptopower root directory and execute the command below to generate a .apk file:
+
+`gogio -target android .`
```

Comment:
I need to do `export ANDROID_SDK_ROOT=$HOME/.local/share/Android/Sdk` which might be useful to mention. Also could mention building with the `--testnet` flag for testnet

Reply:
I also get a prompt to `set ANDROID_SDK_ROOT to the Android SDK path` when I run this command. I have the path set in `$ANDROID_HOME` so I just do `ANDROID_SDK_ROOT=$ANDROID_HOME gogio -target android .`

Reply:
> Also could mention building with the --testnet flag for testnet

good catch

Reply:
I do not have it set but Android Studio runs without it set. There is a clear message when it fails if it is not set though, so it's fine to not mention here I think if you do not want to.

Reply:
Yeah, same here. My setup works without the `ANDROID_SDK_ROOT` env variable. I think that variable name is now `ANDROID_HOME` but gogio still looks for `ANDROID_SDK_ROOT`. Can't remember, but saw something similar somewhere sometime while trying to build with gogio the first time.